src/Bijectors.jl

```diff
@@ -448,4 +449,6 @@ include("interface.jl")
 
 include("norm_flows.jl")
 
+include("norm.jl")
```

Comment:
Not sure about this filename :/ But I'm very much in favour of restructuring a bit of the project, e.g. `src/bijectors/` is a directory containing all bijectors, and then have `batch_norm.jl` be in this. So maybe we can delay the file-renaming until we do the full restructuring